### PR TITLE
mes-9053&9059-alignmentFix

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Corrected icons and alignment on debrief page & fixed alignment on orientation lock icons
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image
image
image

### CODE REVIEW

File: src/app/pages/debrief/components/safety-and-balance-card/safety-and-balance-card.cat-a-mod2.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
   </ion-card-header>
   <ion-card-content>
     <ion-grid>
-      <ion-row class="ion-align-items-center mes-data" *ngFor="let question of safetyAndBalanceQuestions$| async">
+      <ion-row class="ion-align-items-center row-padding mes-data" *ngFor="let question of safetyAndBalanceQuestions$| async">
         <ion-col size="32" class="ion-no-padding"></ion-col>
-        <ion-col size="3" class="down-padding ion-no-padding ion-align-self-start">
-          <div class="counter-icon">
+        <ion-col size="auto" class="ion-no-padding">
```
Comment (andrewsetterfield) [inline src/app/pages/debrief/components/safety-and-balance-card/safety-and-balance-card.cat-a-mod2.html]: auto sounds dangeroly like letting it decide if we want it by itself.  do we not want to control this?

Comment (RLCorp) [reply src/app/pages/debrief/components/safety-and-balance-card/safety-and-balance-card.cat-a-mod2.html]: resolved in newest commit

File: src/app/pages/debrief/components/safety-questions-card/__mocks__/safety-questions-card.mock.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/common';
+
+export const getMalformedVehicleChecks = (): QuestionResult[] => {
```
Comment (matthew2564) [inline src/app/pages/debrief/components/safety-questions-card/__mocks__/safety-questions-card.mock.ts]: Where is this used, and why is it needed?

File: src/app/pages/debrief/debrief.page.html [hunk 3]
```diff
@@ -1,100 +1,94 @@
 <ion-header>
```
Comment (matthew2564) [inline src/app/pages/debrief/debrief.page.html]: Not sure why the formatting has changed so much, as this goes against the `indent_size = 2` on the `.editorconfig`

File: src/app/pages/debrief/debrief.page.html [hunk 4]
```diff
@@ -1,100 +1,94 @@
...
+            [overallScore]="pageState.overallScore$ | async"
+            [question1]="pageState.question1$ | async"
+            [question2]="pageState.question2$ | async"
+            [question3]="pageState.question3$ | async"
+            [question4]="pageState.question4$ | async"
+            [question5]="pageState.question5$ | async"
+            [testOutcome]="outcome">
+    </cpc-debrief-card>
 
+    <safety-questions-card
```
Comment (matthew2564) [inline src/app/pages/debrief/debrief.page.html]: Is this missing the check that used to happen? i.e. `*ngIf="isCatD() && pageState.showSafetyQuestions$ | async"`

File: src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.html [hunk 5]
```diff
@@ -1,36 +1,26 @@
...
-            <ion-col no-padding>
-              <div class="mes-data large">{{item.description}}</div>
-            </ion-col>
-          </ion-row>
+        <ion-grid>
+            <ion-row *ngFor="let item of data" class="mes-data ion-align-items-center">
+                <ion-col
+                        id="icon-column"
+                        class="display-flex ion-align-items-center ion-no-padding completed-icon-padding"
+                        size="9.1">
```
Comment (matthew2564) [inline src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.html]: Personally I feel quite uneasy about using non-integer values for column sizes, I can't recall anywhere else in the app we have needed to do this.

File: src/app/pages/debrief/components/safety-questions-card/__mocks__/safety-questions-card.mock.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/debrief/components/safety-questions-card/__mocks__/safety-questions-card.mock.ts]: resolved in newest commit

File: src/app/pages/debrief/debrief.page.html (hunk 4 above)
Comment (RLCorp) [reply src/app/pages/debrief/debrief.page.html]: resolved in newest commit

File: src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.html (hunk 5 above)
Comment (RLCorp) [reply src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.html]: resolved in newest commit

File: src/app/pages/debrief/components/safety-questions-card/safety-questions-card.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
+<ion-card id="vehicle-checks" *ngIf="data.length > 0; let questions">
```
Comment (matthew2564) [inline src/app/pages/debrief/components/safety-questions-card/safety-questions-card.html]: Can we update to `data?.length`, I've seen a few Sentry errors for similar instances before that we have recently solved.

Comment (RLCorp) [reply src/app/pages/debrief/components/safety-questions-card/safety-questions-card.html]: addressed in newest commit

**Review by matthew2564 — APPROVED** [review]

**Review by andrewsetterfield — APPROVED** [review]

File: src/app/pages/debrief/debrief.page.html (hunk 3 above)
Comment (RLCorp) [reply src/app/pages/debrief/debrief.page.html]: fixed on my end

**Pull request merged**